canary/script/changeCanaryPackageVersion.mjs

```diff
@@ -7,6 +7,13 @@ import { globbyStream } from 'globby';
 import fs from 'fs';
```

Comment:
This change is cherry-picked from https://github.com/aws-amplify/amplify-ui/pull/2768, which enables canary to target any dist-tag. No code changes were made.